cli/tests/037_fetch_multiple.out

```diff
@@ -0,0 +1,6 @@
+Compile file:///home/yamalight/projects/bxjs/deno/cli/tests/fetch/test.ts
+Download https://unpkg.com/leaflet/dist/leaflet-src.esm.js
+Download https://unpkg.com/leaflet@1.6.0/dist/leaflet-src.esm.js
+Compile file:///home/yamalight/projects/bxjs/deno/cli/tests/fetch/other.ts
```

Comment:
These absolute paths will not work on CI, try this:
```
Compile file:///home/yamalight/projects/bxjs/deno/cli/tests/fetch/other.ts

Compile [WILDCARD]/fetch/other.ts
```

Reply:
how did I miss this 🤦‍♂ will fix in a second!

Reply:
And done!